gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/resources/organizations/ReporterResource.java

```diff
@@ -0,0 +1,162 @@
+                .andThen(organizationService.findById(organizationId)
+                        .onErrorResumeWith(Single.error(new OrganizationNotFoundException(organizationId)))
+                        .flatMapMaybe(irrelevant -> reporterService.findById(reporterId))
+                        .switchIfEmpty(Maybe.error(new ReporterNotFoundException(reporterId)))
+                        .map(reporter -> {
+                            if (!reporter.getReference().matches(ReferenceType.ORGANIZATION, organizationId)) {
+                                throw new BadRequestException("Reporter does not belong to organization");
+                            }
+                            return Response.ok(reporter.apiRepresentation(false)).build();
+                        }))
+                .subscribe(response::resume, response::resume);
+    }
+
+    @PUT
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(summary = "Update a reporter",
+            description = "User must have the DOMAIN_REPORTER[UPDATE] permission on the specified domain " +
+                    "or DOMAIN_REPORTER[UPDATE] permission on the specified environment " +
+                    "or DOMAIN_REPORTER[UPDATE] permission on the specified organization")
+    @ApiResponses({
+            @ApiResponse(responseCode = "201", description = "Reporter successfully updated",
+                    content = @Content(mediaType = "application/json",
+                            schema = @Schema(implementation = Reporter.class))),
+            @ApiResponse(responseCode = "500", description = "Internal server error")})
+    public void update(
+            @PathParam("organizationId") String organizationId,
+            @PathParam("reporterId") String reporterId,
+            @Parameter(name = "reporter", required = true) @Valid @NotNull UpdateReporter updateReporter,
+            @Suspended final AsyncResponse response) {
+        final User authenticatedUser = getAuthenticatedUser();
+
+        checkPermission(ReferenceType.ORGANIZATION, organizationId, Permission.ORGANIZATION_REPORTER, Acl.UPDATE)
+                .andThen(organizationService.findById(organizationId)
+                        .onErrorResumeWith(Single.error(new OrganizationNotFoundException(organizationId)))
+                        .flatMap(__ -> reporterService.update(Reference.organization(organizationId), reporterId, updateReporter, authenticatedUser, false)))
+                .subscribe(response::resume, response::resume);
+    }
+
+    @DELETE
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(summary = "Delete a reporter",
+            description = "User must have the DOMAIN_REPORTER[DELETE] permission on the specified domain " +
+                    "or DOMAIN_REPORTER[DELETE] permission on the specified environment " +
+                    "or DOMAIN_REPORTER[DELETE] permission on the specified organization")
```

Comment:
**TODO:** permission shouldn't be related to the domain or env here as we are at Org level